src/Features/CSharp/Portable/Diagnostics/Analyzers/CSharpTypeStyleDiagnosticAnalyzerBase.cs

```diff
@@ -62,9 +58,7 @@ private void HandleVariableDeclaration(SyntaxNodeAnalysisContext context)
 
             var typeStyle = Helper.AnalyzeTypeName(
                 declaredType, semanticModel, optionSet, cancellationToken);
-            if (!typeStyle.IsStylePreferred ||
```

Comment:
@sharwell what process did you use to figure out waht would need updating?  did you basically just look for usages of .Hidden?

Reply:
Basically, i'm trying to figure out if we got everything, or if any analyzers might have slipped through.

Reply:
I looked for uses of `Hidden` and also accounted for the diff of #27220. The type change there did a good job of highlighting relevant cases.